### ISSUE TITLE
TypeError thrown when running in minimal mode

### ISSUE THREAD

**Issue by TropicalRaisel** [FormidableLabs/webpack-dashboard#336]
> Please provide a description of the bug / issue, and provide the details below:

When running this command: `webpack-dashboard -m -- encore dev-server`, this error gets generated after a successful build:
```
TypeError: Cannot read property 'type' of null
    at Socket.<anonymous> (C:\...\node_modules\webpack-dashboard\bin\webpack-dashboard.js:79:21)
    at Socket.emit (node:events:394:28)
    at Socket.emitUntyped (C:\...\node_modules\socket.io\dist\typed-events.js:69:22)
    at C:\...\node_modules\socket.io\dist\socket.js:428:39
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
error Command failed with exit code 1.
```

---

###### If the issue is visual, please provide screenshots here

---

###### Steps to reproduce the problem

1. Clone my repo: https://github.com/TropicalRaisel/django-encore-webapp
2. Go into the `package.json` and change the first script to the command shown in the description above.
3. `yarn serve`

---

###### Please provide a gist of relevant files

1. [package.json](https://github.com/TropicalRaisel/django-encore-webapp/blob/main/package.json#L10) (specifically the script you are using to start the dashboard)
2. [webpack.config.js](https://github.com/TropicalRaisel/django-encore-webapp/blob/main/webpack.config.js)

---

###### More Details

- What operating system are you on? Windows 10
- What terminal application are you using? Tested on Command Prompt and Powershell
- What version of webpack-dashboard are you using? `^3.3.5`
- What is the output of running `echo $TERM`? Nothing

**Comment by zicodeng** [comment]
I believe this is a duplicate of this [issue](https://github.com/FormidableLabs/webpack-dashboard/issues/335) 🤔

**Comment by ryan-roemer** [comment]
Fixed in `webpack-dashboard@3.3.6`
